### PR TITLE
#1543 - Update designation form content

### PR DESCRIPTION
**Pull request by hellolynn-tbtb**
### Updates
- Move text underneath heading
- Content change on bullet points

image

### Figma (for reference only)
- https://www.figma.com/file/9Om2tzLAVEoMQrjmrH4fhu/MOD-SIMS?node-id=9474%3A97285&t=jEspGEbtvSsanZjp-0

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/designationagreementdetails.json [hunk 1]
```diff
@@ -1,12 +1,14 @@
 {
+  "_id": "639baa9cc303847c56c151c4",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/designationagreementdetails.json:2]: `"_id"` and `"owner"` can be removed.

**Review by andrewsignori-aot — COMMENTED** [review]
Just a minor comment 😉

**Review by ann-aot — COMMENTED** [review]
Good work @hellolynn-tbtb 👍I will approve when signori's comment is addresed

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the change, look good 👍

**Pull request merged**